home/gettingstarted/0.7/executors.md

```diff
@@ -155,9 +155,40 @@ And the final output:
         (ci = <20000.02 kB, 20000.02 kB>, significance = 1.0E-10)
 
 
-You can take a look at the ScalaMeter API or source code to figure out how different measurers work in more detail.
+### Method Invocation Count
+
+The invocation count measurers are used to count the total number of invocations
+of a specific method, or a group of methods.
+
+There are two implementations of the `InvocationCount` trait and both of them need instrumented classpath to work,
+hence `SeparateJvmsExecutor` must be used as an executor.
+
+`BoxingCount` simply counts (auto)boxing of primitives: `Boolean`, `Byte`, `Char`, `Short`, `Int`, `Long`, `Float` and `Double`.
+You can either specify primitives you would like to count or count all by using `BoxingCount.all()` method.
+
+`MethodInvocationCount` can count invocations of arbitrary methods. Configuration of it is done via specifying `InvocationCountMatcher` constructor argument.
```

Comment:
Please give another minimal example of instantiating this measurer, e.g.:

    override def measurer = MethodInvocationCount(InvocationCountMatcher.forName(..., ...))